### PR TITLE
Fix doc style

### PR DESCRIPTION
**Pull request by maxcanna**
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by maxcanna** [comment]
I'm trying to fix this rendering problem in the doc

Schermata 2020-07-09 alle 10 43 18

File: source/_integrations/google_assistant.markdown [hunk 1]
```diff
@@ -149,7 +149,7 @@ entity_config:
   required: false
   type: map
   keys:
-    '`<ENTITY_ID>`':
+    `'<ENTITY_ID>'`:
```
Comment (frenck) [inline source/_integrations/google_assistant.markdown]: As a workaround:

```suggestion
    YOUR_ENTITY_ID:
```

**Comment by maxcanna** [comment]
Thank you for the quick feedback on this @frenck . I wanted to do like you proposed but didn't know if it'd be good. Lmk

**Review by frenck — APPROVED** [review]
Thanks, @maxcanna 👍

**Pull request merged**